packages/@aws-cdk/aws-stepfunctions-tasks/test/emrcontainers/start-job-run.test.ts

```diff
@@ -0,0 +1,836 @@
+    virtualClusterId: sfn.TaskInput.fromText(clusterId),
+    releaseLabel: ReleaseLabel.EMR_6_2_0,
+    jobDriver: {
+      sparkSubmitJobDriver: {
+        entryPoint: sfn.TaskInput.fromText('job-location'),
+      },
+    },
+    integrationPattern: sfn.IntegrationPattern.RUN_JOB,
+  });
+
+  new sfn.StateMachine(stack, 'SM', {
+    definition: task,
+  });
+
+  // THEN
+  expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+    PolicyDocument: {
+      Statement: [{
+        Action: 'emr-containers:StartJobRun',
+        Condition: {
+          StringEquals: {
+            'emr-containers:ExecutionRoleArn': {
+              'Fn::GetAtt': [
+                'TaskJobExecutionRole5D5BBA5A',
+                'Arn',
+              ],
+            },
+          },
+        },
+        Resource: {
+          'Fn::Join': [
+            '',
+            [
+              'arn:',
+              {
+                Ref: 'AWS::Partition',
+              },
+              ':emr-containers:',
+              {
+                Ref: 'AWS::Region',
+              },
+              ':',
+              {
+                Ref: 'AWS::AccountId',
+              },
+              `:/virtualclusters/${clusterId}`,
+            ],
+          ],
+        },
+      },
+      {
+        Action: [
+          'emr-containers:DescribeJobRun',
+          'emr-containers:CancelJobRun',
+        ],
+        Effect: 'Allow',
+        Resource: {
+          'Fn::Join': [
+            '',
+            [
+              'arn:',
+              {
+                Ref: 'AWS::Partition',
+              },
+              ':emr-containers:',
+              {
+                Ref: 'AWS::Region',
+              },
+              ':',
+              {
+                Ref: 'AWS::AccountId',
+              },
+              `:/virtualclusters/${clusterId}/jobruns/*`,
+            ],
+          ],
+        },
+      }],
+    },
+  });
+
+  expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+    PolicyDocument: {
+      Statement: [
+        {
+          Action: [
+            'logs:DescribeLogGroups',
+            'logs:DescribeLogStreams',
+          ],
+          Effect: 'Allow',
+          Resource: 'arn:aws:logs:*:*:*',
+        },
+      ],
+      Version: '2012-10-17',
+    },
+    PolicyName: 'TaskJobExecutionRoleDefaultPolicy6FE2FE84',
+    Roles: [
+      {
+        Ref: 'TaskJobExecutionRole5D5BBA5A',
+      },
+    ],
+  });
+
+  expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+    PolicyDocument: {
+      Statement: [{
+        Action: 'emr-containers:DescribeVirtualCluster',
+        Resource: '*',
+      }],
+    },
+  });
+
+  expect(stack).toHaveResourceLike('AWS::IAM::Policy', {
+    PolicyDocument: {
+      Statement: [{
+        Action: 'lambda:InvokeFunction',
+        Resource: {
+          'Fn::GetAtt': [
+            'SingletonLambda8693BB64968944B69AAFB0CC9EB8757CB6182A5B',
+            'Arn',
+          ],
+        },
+      }],
+    },
+  });
```

Comment:
these permissions are unrelated to the state machine execution role and should not be in this test